content/livesync/models/models.textile

```diff
@@ -178,7 +170,7 @@ The @model.optimistic()@ function returns a promise resolving to two values:
 * A confirmation promise that resolves when the backend confirms or rejects the optimistic update.
 * A function to explicitly cancel the optimistic update.
 
-The following is an example of optimistically applying changes to a model:
+The following demonstrates an optimistic update to implementing changes in the model:
```

Comment:
>optimistic update to implementing

Not sure this is reads quite right, could we say e.g. "The following is an example of an optimistic update to the model state"